app/src/main/java/com/github/geohunt/app/ui/components/cards/ChallengeCard.kt

```diff
@@ -0,0 +1,263 @@
+package com.github.geohunt.app.ui.components.cards
+
+import androidx.compose.animation.core.Spring
+import androidx.compose.animation.core.animateDpAsState
+import androidx.compose.animation.core.spring
+import androidx.compose.foundation.gestures.detectTapGestures
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Logout
+import androidx.compose.material.icons.filled.PersonAdd
+import androidx.compose.material.icons.filled.PersonRemove
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.github.geohunt.app.R
+import com.github.geohunt.app.i18n.DateFormatUtils
+import com.github.geohunt.app.model.Challenge
+import com.github.geohunt.app.model.Location
+import com.github.geohunt.app.model.User
+import com.github.geohunt.app.ui.components.buttons.ChallengeHuntState
+import com.github.geohunt.app.ui.components.buttons.HuntClaimButton
+import com.github.geohunt.app.ui.components.buttons.MenuButton
+import com.github.geohunt.app.ui.components.buttons.MenuItem
+import com.github.geohunt.app.ui.components.buttons.OpenMapButton
+import com.github.geohunt.app.ui.components.utils.SkeletonLoading
+import com.github.geohunt.app.ui.components.utils.SkeletonLoadingImage
+import com.github.geohunt.app.ui.theme.geoHuntRed
+import com.github.geohunt.app.utility.quantizeToLong
+import java.time.LocalDateTime
+
+val ChallengeCardContentPadding = 8.dp
```

Comment:
Can't this value go somewhere else ?

Reply:
This is a constant that is related to the UI of the cards, don't know where we could put it otherwise.